src/components/molecules/Settings/SettingPage/index.tsx

```diff
@@ -2,12 +2,20 @@ import { Link } from "@reach/router";
 import React, { useState } from "react";
 
 import Icon from "@reearth/components/atoms/Icon";
+import Loading from "@reearth/components/atoms/Loading";
 import Header, { Props } from "@reearth/components/molecules/Common/Header";
 import ProjectMenu from "@reearth/components/molecules/Common/ProjectMenu";
 import Navigation from "@reearth/components/molecules/Settings/Navigation";
 import { styled } from "@reearth/theme";
+import { handleScroll } from "@reearth/util/handleScroll";
 
-const SettingPage: React.FC<Props> = ({
+interface SettingPageProps extends Props {
+  loading?: boolean;
+  hasMoreItems?: boolean;
+  onScroll?: () => void;
+}
```

Comment:
Please doublecheck that my suggestion's syntax is correct. I might have something mixed up a little
```suggestion
export type SettingPageProps = {
  loading?: boolean;
  hasMoreItems?: boolean;
  onScroll?: () => void;
} & Props
```

Reply:
Also, below where you are declaring the props, please destructure out the `loading`, `hasMoreItems`, and `onScroll from props.
You use them here in this component, so it would be nicer to not have to write `props.hasMoreItems`. But also, you are spreading props into the StyledHeader, which means currently you are also spreading into the Header the above props (which I believe the Header does not need or want).